### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [twireapp/Twire#254] [General]
Would a Mastodon account for this project be possible?

**Comment by samfundev** [comment]
I don't think we have any social media accounts for Twire, so this is a bit of unexplored territory for Twire. I assume that the reason we would be creating this would be to promote ourselves or keep up with changes, but perhaps @Lukas2112 had something else in mind.

**Reply by linux-lukas** [reply]
The idea behind a Twire-Mastodon account would be to publish project announcements and project information in general. In addition, a active interaction with the Fediverse community would be possible.

Two positive side effects that a project mastodon account would bring would be 1. that the Twire project would become better known and 2. people who have been recommended Twire by people in their IT circle of friends or acquaintances would be indirectly pointed to privacy-friendly social network alternatives.

@samfundev

**Reply by Perflyst** [reply]
In my opinion Twire is way too small and also someone has to manage the Mastodon account. It might be something for the future, but I don't think we need "social media" at the current state.
